scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -1498,6 +1498,13 @@ boolean changeStartAt(JobId jobId, String startAt)
     JobId copyJobAndResubmitWithGeneralInfo(JobId jobId, Map<String, String> generalInfo) throws NotConnectedException,
             UnknownJobException, PermissionException, SubmissionClosedException, JobCreationException;
 
+    /**
+     * @param jobId job id of existing job
+     * @return copy of the object which was submitted to the scheduler
+     */
+    Job getJobContent(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)